src/js/components/InfiniteScroll/stories/Replace.js

```diff
@@ -24,10 +24,20 @@ const InfiniteScrollReplace = props => (
 if (!isChromatic()) {
   storiesOf('InfiniteScroll', module)
     .add('Replace', () => <InfiniteScrollReplace replace />)
-    .add('Replace with show 28th item', () => (
-      <InfiniteScrollReplace replace show={27} />
-    ))
-    .add('Replace with show 88th item', () => (
-      <InfiniteScrollReplace replace show={87} />
+    .add('Replace with Show', () => (
```

Comment:
```suggestion
    .add('Replace with show', () => (
```
We try to use sentence case on stories